library/core/src/test/java/com/google/android/exoplayer2/text/ttml/TtmlDecoderTest.java

```diff
@@ -674,6 +676,105 @@ public void rubies() throws IOException, SubtitleDecoderException {
     assertThat(sixthCue).hasNoRubySpanBetween(0, sixthCue.length());
   }
 
+  @Test
+  public void textEmphasis() throws IOException, SubtitleDecoderException {
+    TtmlSubtitle subtitle = getSubtitle(TEXT_EMPHASIS_FILE);
+
+    Spanned firstCue = getOnlyCueTextAtTimeUs(subtitle, 10_000_000);
+    assertThat(firstCue)
+        .hasNoTextEmphasisSpanBetween("None ".length(), "None おはよ".length());
+
+    Spanned secondCue = getOnlyCueTextAtTimeUs(subtitle, 20_000_000);
+    assertThat(secondCue)
+        .hasTextEmphasisSpanBetween("Auto ".length(), "Auto ございます".length())
+        .withMarkAndPosition(TextEmphasisSpan.MARK_FILLED_CIRCLE,
+            TextEmphasisSpan.POSITION_UNKNOWN);
```

Comment:
Shouldn't this be `POSITION_OUTSIDE`?

>If no emphasis position is specified, then the emphasis position must be interpreted as if a position of outside were specified.

https://www.w3.org/TR/2018/REC-ttml2-20181108/#style-attribute-textEmphasis

Reply:
Fixing.